src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -583,14 +611,13 @@ private static void downloadAndSetupDist(PrintStream printStream, HttpURLConnect
             }
 
             new File(zipFileLocation).delete();
-        } finally {
-            conn.disconnect();
         }
     }
 
-    public static void getDependency(PrintStream printStream, String distribution, String distributionType,
+    public static String getDependency(PrintStream printStream, String distribution, String distributionType,
                                      String distributionVersion) {
         HttpsURLConnection conn = null;
+        String dependencyName = "";
         try {
             printStream.println("\nFetching the dependencies for '" + distribution + "' from the remote server...");
```

Comment:
Shall we remove this from here as we are not fetching dependency in this method?
We can move this when we download the dependency

Reply:
In this method (`getDependency`) what we do is getting the dependency name for a given distribution version.